ecommerce/src/Relecloud.FunctionApp/EventProcessor/EventProcessorFunction.cs

```diff
@@ -1,170 +1,214 @@
-﻿using Microsoft.Azure.WebJobs;
-using Microsoft.Azure.WebJobs.Host;
-using Microsoft.WindowsAzure.Storage.Blob;
-using Newtonsoft.Json;
-using System;
-using System.Configuration;
-using System.Data.SqlClient;
+using Azure;
+using Azure.AI.TextAnalytics;
+using Azure.Storage.Blobs;
+using Azure.Storage.Sas;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Data.SqlClient;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
 using System.Drawing;
 using System.Drawing.Drawing2D;
 using System.Drawing.Imaging;
-using System.IO;
-using System.Net.Http;
-using System.Text;
-using System.Threading.Tasks;
+using System.Runtime.InteropServices;
 
 namespace Relecloud.FunctionApp.EventProcessor
 {
-    public static class EventProcessorFunction
+    public class EventProcessorFunction
     {
-        #region Run
+        private readonly ILogger _logger;
+        private readonly IConfiguration _configuration;
 
-        [FunctionName("EventProcessor")]
-        public static async Task Run(
-            [QueueTrigger("%App:StorageAccount:EventQueueName%", Connection = "App:StorageAccount:ConnectionString")]Event eventInfo,
-            [Blob("tickets/ticket-{EntityId}.png", FileAccess.ReadWrite, Connection = "App:StorageAccount:ConnectionString")]CloudBlockBlob ticketImageBlob,
```

Comment:
Any reason the blob couldn't be written as an output binding anymore? Not a huge deal of course but there's something to be said for the simplicity of dealing with only the output rather than interacting with the Storage SDK.